robyn/reloader.py

```diff
@@ -4,17 +4,15 @@
 import subprocess
 import sys
 import time
-from typing import List
+from typing import List, Union
 
 from watchdog.events import FileSystemEventHandler
 from watchdog.observers import Observer
 
 from robyn.logger import Colors, logger
 
-dir_path = None
 
-
-def compile_rust_files(directory_path: str):
+def compile_rust_files(directory_path: str) -> list[str]:
```

Comment:
Hey @dave42w 👋 

This is not a timeout 😅 
```suggestion
def compile_rust_files(directory_path: str) -> List[str]:
```

Reply:
Oops fixed